src/translator/parse/common/parse-base-schema.ts

```diff
@@ -9,7 +9,8 @@ import { TElementSchema } from "../types";
 export const parseBaseSchema = (
     schema: TElementSchema,
     id: string,
-    prefill: IPrefillConfig
+    prefill: IPrefillConfig,
+    defaultValue: string | undefined
```

Comment:
can declare it as optional, you will need to update the typing for all other `defaultValue` in the functions calling it too

```suggestion
    defaultValue?: string
```